.github/workflows/aws-deploy.yaml

```diff
@@ -45,7 +45,9 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Generate AWS CDK bootstrap template
-        run: npm run generate-cdk-bootstrap-template
+        run: npx cdk bootstrap --json --show-template > org-formation/200-baseline/tmptemplate.json
+      - name: Add EnableKeyRotation property to CDK assets key
+        run: jq '.Resources.FileAssetsBucketEncryptionKey.Properties += {"EnableKeyRotation" :true}' org-formation/200-baseline/tmptemplate.json > org-formation/200-baseline/cdk-bootstrap.json
```

Comment:
much nicer than the `sed` before 😄 .  please add `cdk-bootstrap.json` to `.gitignore` as well.